content/en/blog/2024/otel-collector-anti-patterns/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: OpenTelemetry Collector Antipatterns
+linkTitle: OTel Collector Antipatterns
+date: 2024-02-23
```

Comment:
```suggestion
date: 2024-03-01
```

Let's have this on Friday, since we will have the KubeCon blog post first, wdyt?